charts/intel-gpu-resource-driver/values.yaml

```diff
@@ -9,7 +9,7 @@ image:
   repository: intel
   name: intel-gpu-resource-driver
   pullPolicy: IfNotPresent
-  tag: "v0.6.0"
+  tag: "v0.7.0"
```

Comment:
Are we keeping Chart version same as DRA driver release version? If so, please, bump version in Chart.yaml